src/app/game.ts

```diff
@@ -343,35 +343,40 @@ export class Game {
     }
   }
 
+  private increaseResourcesNeededToBuild(builing: Building) {
```

Comment:
Would it be possible to move this function to Building class and call it there after the building was built?

Reply:
Indeed it is, done